claims.go

```diff
@@ -61,7 +61,7 @@ func (c RegisteredClaims) Valid() error {
 	}
 
 	if !c.VerifyIssuedAt(now, false) {
-		vErr.Inner = fmt.Errorf("Token used before issued")
+		vErr.Inner = fmt.Errorf("token used before issued")
```

Comment:
Curious, why this PR? Are you relying on the capitilization?

I'm indifferent about this one way or the other, but I'd prefer to avoid these kinds of PRs. Closing.

Also. https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Didn't he try to *remove* the capitalisation? It seems that we missed this one, the other `Errorf` calls are also lowercased.

Reply:
Oops, diff'd it wrong. You're right, apologies @YoogoC . Thanks for your contribution. My mistake.